packages/functional-tests/pages/login.ts

```diff
@@ -219,6 +221,34 @@ export class LoginPage extends BaseLayout {
     return this.page.fill(selectors.PASSWORD, password);
   }
 
+  async showPasswordMouseAction() {
+    return Promise.all([
+      await this.page.locator(selectors.PASSWORD).type('password'),
+      await this.page.locator(selectors.SHOW_PASSWORD).hover(),
+      await this.page.mouse.down(),
+    ]);
+  }
+
+  async showPassword() {
+    return this.page.locator(selectors.SHOW_PASSWORD).isVisible();
+  }
+
+  async typeText() {
+    const pass = this.page.locator('#password[type=text]');
+    await pass.waitFor({
+      state: 'visible',
+    });
+    return pass.isVisible();
+  }
+
+  async typePassword() {
```

Comment:
This function name and the one above are a little confusing.  It sounds like they involve typing something into inputs.  Can you name them something like isXyzVisible?